crates/rpc/ipc/src/client.rs

```diff
@@ -65,7 +85,10 @@ impl TransportSenderT for Sender {
 /// Receiving end of IPC transport.
 #[derive(Debug)]
 pub struct Receiver {
-    inner: FramedRead<tokio::net::unix::OwnedReadHalf, StreamCodec>,
+    #[cfg(unix)]
+    inner: FramedRead<OwnedReadHalf, StreamCodec>,
+    #[cfg(windows)]
+    inner: FramedRead<Arc<NamedPipeClient>, StreamCodec>,
```

Comment:
this is great,

what I would like to do instead is:
convert client into a module with
unix.rs
win.rs

and duplicate the types then client/mod.rs reexports based on windows or UNIX